### DISCUSSION TITLE
Zones... Dont get it...

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#2318] [Ask A Question]
Hi... Using 0.10-beta4...
I've 3 zones in my parking area...
When people/cars are deteced at the road outside my parking zone frigate start to notifiy me about a new object at my parking zone. 
Why does this happen? I suspect that this have to do with "stationary_objects" ... But anyway... My parking zone should not be activated because there isnt any motion in that area. 
![image](https://user-images.githubusercontent.com/58695742/143215980-4ed7c6bf-c66e-45e5-b372-377b2b24882e.png)

**Comment by blakeblackshear** [comment]
You are most likely experiencing something like this: https://github.com/blakeblackshear/frigate/issues/1544

**Reply by Minglarn** [reply]
It seems like it...
The region covers the whole area for a couple of frames and goes to only surround the moving object.. and goes wide and big again... 
Not sure how to minimize this... 

![image](https://user-images.githubusercontent.com/58695742/143482974-c8a1002a-be0e-4382-a142-dea1ed7d44e1.png)

**Reply by blakeblackshear** [reply]
You can see the blue bounding box is massive in that image. I can't quite read the label, but you should set a max_area filter for that type to prevent it. The region grows because the bounding box is so big and its looking for the same object again.

**Reply by Minglarn** [reply]
Allright... Will add max_area to label "motorfordon" (aka car) and see if that makes any difference...
